File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;

        String oddStr = "";
        String evenStr = "";

        for (int i = 0; i < num_list.length; i++) {
            if (num_list[i] % 2 == 0) {
                evenStr += num_list[i];
            } else {
                oddStr += num_list[i];
            } 
        }

        int oddNum = oddStr.isEmpty() ? 0 : Integer.parseInt(oddStr);
        int evenNum = evenStr.isEmpty() ? 0 : Integer.parseInt(evenStr);
        
        answer = oddNum + evenNum;
        
        return answer;
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    answer = 0

    a = ""
    b = ""

    for strNum in num_list:
        if strNum % 2 == 1:
            a += str(strNum)
        elif strNum % 2 == 0:
            b += str(strNum)

    answer = int(a) + int(b)

    return answer